mathml/relations/html5-tree/css-inline-style-interface.tentative.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>MathML 'ElementCSSInlineStyle` Mixin Tests</title>
+    <link
+      rel="help"
+      href="https://mathml-refresh.github.io/mathml-core/#dom-and-javascript"
+    />
+    <style>
+        math * { background-color: red; }
+    </style>
+    <meta name="assert" content="MathMLElements incorporate a functional ElementCSSInlineStyle interface" />
+    <script src="/mathml/support/mathml-fragments.js"></script>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+
+  </head>
+  <body>
+    <span>This tests the presence and functionality of features of the `ElementCSSInlineStyle` interface for MathMLElements</span>
+    <math>
+      <mi>E</mi>
+    </math>
+  </body>
```

Comment:
This should be moved at the end of the file.

Reply:
Why so?

Reply:
Is it valid to have script tag after the body?

Reply:
I mean, its fine to not even close the body, but, I do this all the time because then it doesn't wind up as a random child of the body (using the last child of the body is often valuable if you need to manage 'top layers'), and you know that the body is done.  If it really bothers you I guess I can change it?

Reply:
I think what other tests do to have clear separation between scripts and body is to put all the scripts in the head and starting when the page load event is fired. But whatever you want, my concern was whether it is valid to have as script after the body, I never do that so I don't know. Personally, I don't like no closing tags, as that makes harder to understand for people reading the tests to know when they are actually closing.

Reply:
I moved this inside body, does that work?